### DISCUSSION TITLE
Is there a way to identify the cell at which a existing chart is placed in the worksheet? Current we can only add charts but is there a facility to read chart properties?

### DISCUSSION THREAD

**Discussion by guruRockz** [qax-os/excelize#1930] [Q&A]
Is there a way to identify the cell at which a existing chart is placed in the worksheet? Current we can only add charts but is there a facility to read chart properties?. Looking for a requirement to read the position of a chart and then relocate it to a different predefined cell.

**Comment by xuri** [comment]
Thanks for your feedback. This library doesn't support get charts currently. I will consider add support for this later.
